src/clib.jl

```diff
@@ -1,69 +0,0 @@
-# C functions in the library
-
-@unix_only const libxml2 = Libdl.dlopen("libxml2", Libdl.RTLD_GLOBAL)
-@windows_only const libxml2 = Libdl.dlopen(Pkg.dir("WinRPM","deps","usr","$(Sys.ARCH)-w64-mingw32","sys-root","mingw","bin","libxml2-2"))
-
-macro lx2func(fname)  # the macro to get functions from libxml2
-    quote
-        $(esc(fname)) = Libdl.dlsym( libxml2, ($(string(fname))) )
-    end
-end
-
-const nullptr = convert(Ptr{Void}, 0)
-const ptrsize = sizeof(Ptr{Void})
-
-@assert ptrsize == sizeof(Uint)
-
-typealias Xchar Uint8
-typealias Xstr Ptr{Xchar}
-typealias Xptr Ptr{Void}
-
-# supporting functions
-
-#
-# After tests, it seems that free in libc instead of xmlFree
-# should be used here
```

Comment:
This looks like a comment I'd like to keep for future references. Ref https://github.com/JuliaLang/LightXML.jl/commit/b33d030cb052f05d528a7355f2237471103a394c @lindahua